rt/frontend/jaxrs/src/main/java/org/apache/cxf/jaxrs/model/doc/JavaDocProvider.java

```diff
@@ -74,12 +83,12 @@ public JavaDocProvider(Bus bus, String... paths) throws Exception {
         }
     }
 
-    private static double getVersion() {
+    protected static double getVersion() {
```

Comment:
Could we keep this methods protected or there is a reason to open them up?

Reply:
I tend to open up all methods for classes that might be extended by 3rd party apps; being also `static`, moreover, there should be no danger with override.
Anyway, I've reverted this.